frontend_next/app/(root)/case-studies/page.tsx

```diff
@@ -0,0 +1,43 @@
+import Image from "next/image";
+
+import { caseStudies } from "@/constants";
+
+export default function CaseStudies() {
+  return (
+    <>
+      <section className="bg-1 text-center">
+        <h1 className="pageTitle">Recent Case Studies</h1>
+        <p className="pageSubTitle mx-auto max-w-[658px]">
+          Dive into my recent success stories and discover how I have helpe
+          clients overcome challenges, innovate, and achieve their goals
+        </p>
+      </section>
+
+      <section className="bg-2 flex flex-wrap justify-center gap-5 md:gap-9">
+        {caseStudies.map((project) => (
+          <div key={project.title}>
+            <div
+              className={`flex min-h-[205px] items-end rounded-[16px] ${project.bgColor} sm:min-h-[327px] `}
+            >
+              <Image
+                src={project.imagePath}
+                alt="project image"
+                width={0}
+                height={0}
```

Comment:
It's important for the browser when rendering the image to know the size of the images. Have you ever been on a website and as its loading all the elements are shifting around and moving? This can be a cause of that. Set the correct dimensions so the browser knows what space to make for the element as it loads in.